From 860eef7116e21de9e7bc96ac8e06428f35b99f41 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Tue, 11 Mar 2025 02:30:37 +0100
Subject: [PATCH 8/8] ARM: dts: imx: Drop bogus regulator extras on DH i.MX6
 DHCOM DRC02

The regulator extras should be placed in the USB H1 regulator node,
the /regulator-usb-h1-vbus. They are already present there in the
upstream DT, so delete this bogus node entirely.

Upstream-Status: Submitted [https://patchwork.ozlabs.org/project/uboot/patch/20250311013430.510151-1-marex@denx.de/]
Signed-off-by: Marek Vasut <marex@denx.de>
---
 arch/arm/dts/imx6qdl-dhcom-u-boot.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/dts/imx6qdl-dhcom-u-boot.dtsi b/arch/arm/dts/imx6qdl-dhcom-u-boot.dtsi
index 740a24d96ec..300e355c8ef 100644
--- a/arch/arm/dts/imx6qdl-dhcom-u-boot.dtsi
+++ b/arch/arm/dts/imx6qdl-dhcom-u-boot.dtsi
@@ -27,11 +27,6 @@
 	};
 };
 
-&reg_usb_otg_vbus {
-	gpio = <&gpio3 31 GPIO_ACTIVE_HIGH>;
-	enable-active-high;
-};
-
 &wdog1 {
 	bootph-pre-ram;
 };
-- 
2.47.2

